src/dash_tools.jl

```diff
@@ -378,9 +404,26 @@ function make_time_card()
 end
 
 """
-Retunrs a row containing a label, a tooltip and a filling box.
+Returns a row containing a label, a tooltip and a filling box.
 """
-function make_accordion_item(label::String="param", idx::String="id", msg::String="Message", value::Float64=1.0, min::Float64=1.0e-10, max::Float64=10_000.0)
+function make_accordion_item(label::String="param", idx::String="id", msg::String="Message", value::_T=1*one(_T), min=nothing, max=nothing) where _T <: Number
```

Comment:
Perhaps is better using different names for `min` and `max`, they are both julia functions. Maybe `low` and `high`?